portfolio/src/main/java/com/google/DataServlet.java

```diff
@@ -13,20 +13,37 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-
+import java.util.ArrayList;
```

Comment:
Can you keep the imports in the ASCII sort order?
https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing

Reply:
Will do, I'll send the update to the same branch